Utilities/StorageFactory/plugins/HttpStorageMaker.cc

```diff
@@ -18,6 +18,8 @@ namespace edm::storage {
 
       return RemoteFile::get(localfd, temp, (char **)curlopts, mode);
     }
+
+    UseLocalFile usesLocalFile() const override { return UseLocalFile::kYes; }
```

Comment:
I wouldnt' have expected HTTP to be a 'local file'

Reply:
The misnomer is worse than that, this storage implementation is used only for `ftp:` protocol (`http:` and `https:` use the `DavixAdaptor`). Calling this storage implementation e.g. `CurlStorageMaker` would be more accurate. Not sure if the name change would be worth it (maybe, as it is not visible outside of the storage system).